src/libraries/System.Private.CoreLib/src/System/Reflection/FunctionPointerInfo.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Reflection
+{
+    internal sealed partial class FunctionPointerInfo
+    {
+        private const string CallingConventionTypePrefix = "System.Runtime.CompilerServices.CallConv";
+        private readonly RuntimeFunctionPointerParameterInfo _returnInfo;
+        private readonly RuntimeFunctionPointerParameterInfo[] _parameterInfos;
+        private Type[]? _callingConventions;
+
+        public FunctionPointerParameterInfo ReturnParameter => _returnInfo;
+        public FunctionPointerParameterInfo[] ParameterInfos => CloneArray(_parameterInfos);
+
+        public Type[] GetCallingConventions()
+        {
+            if (_callingConventions == null)
+            {
+                Type[] customModifiers = _returnInfo.GetOptionalCustomModifiers();
+                List<Type>? list = null;
+                bool foundCallingConvention = false;
+
+                for (int i = 0; i < customModifiers.Length; i++)
+                {
+                    Type type = customModifiers[i];
+                    if (type.FullName!.StartsWith(CallingConventionTypePrefix))
```

Comment:
```suggestion
                    if (type.FullName!.StartsWith(CallingConventionTypePrefix, StringComparison.Ordinal))
```
`StartsWith` uses current culture by default.

Reply:
Thanks